.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,85 @@
+<!--
+Thank you for your contribution! Before opening this pull request, please complete the template
+completely. Unless instructed otherwise, do not delete any sections.
+-->
+## Description
+<!--
+Describe what this change does and the motivation behind it. Why is it required? What problems does
+it solve?
+-->
+
+## Breaking change
+<!--
```

Comment:
Would we want to add something in this section along the lines of "Is a breaking change required? Could a breaking change be avoided by tolerant deprecation instead?" or something similar.

Reply:
Perfect! 🎉